core/src/main/java/com/airwallex/airskiff/core/api/Stream.java

```diff
@@ -95,6 +96,10 @@ default Stream<T> union(Stream<T> that) {
     return new ConcatStream<>(this, that);
   }
 
+  default Stream<T> union(Stream<T> that, Duration allowedLatency) {
```

Comment:
It does not feel natural/adequate to put allowedLatency in here. What if we make it a global config for all operations?

Reply:
cool, a global config seems more graceful